<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTemperatureDialog</class>
 <widget class="QDialog" name="CTemperatureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>整体</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>布局</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="temArrangeBtn">
          <property name="minimumSize">
           <size>
            <width>93</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>93</width>
            <height>75</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="temHelpBtn">
          <property name="minimumSize">
           <size>
            <width>93</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>93</width>
            <height>75</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>W1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="W1_Edit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="W1_Slider">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>W2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="W2_Edit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="W2_Slider">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>H1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="H1_Edit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="H1_Slider">
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>H2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="H2_Edit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="H2_Slider">
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>偏移量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="offsetEdit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="offsetSlider">
          <property name="minimum">
           <number>-550</number>
          </property>
          <property name="maximum">
           <number>550</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>阻尼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stepEdit">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="stepSlider">
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>表达式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="expEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="expBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>边缘</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="edgeColorBtn">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>样式</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="edgeStyleBtn">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>宽度</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="edgeWidEdit">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="edgeWidSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>温度计填充</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="tempColor1Btn">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="tempColor2Btn">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>渐变</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="tempGradBtn">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_17">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>背景</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_19">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bkColor1Btn">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_20">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bkColor2Btn">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_21">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>渐变</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_22">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bkGradBtn">
          <property name="minimumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_23">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
